crypto/.js/src/main/scala/org/bitcoins/crypto/BCryptoCryptoRuntime.scala

```diff
@@ -246,6 +257,22 @@ trait BCryptoCryptoRuntime extends CryptoRuntime {
             s"Need $len bytes for buffer -> bytevector conversion")
     ByteVector(accum.map(_.toByte))
   }
+
+  override def decodePoint(bytes: ByteVector): ECPoint = {
+    if (bytes.size == 1 && bytes(0) == 0x00) {
```

Comment:
nit: `if (bytes == ByteVector(0))`

Reply:
I believe it's to much of overhead for a one of the most used methods to allocate at least 21 bytes just to check if one byte is set zero.